cli/internal/login/link.go

```diff
@@ -237,6 +235,51 @@ func promptSetup(location string) (bool, error) {
 	return shouldSetup, nil
 }
 
+func (l *link) addTurboToGitignore() error {
+	gitignorePath := l.base.RepoRoot.Join(".gitignore")
+
+	err := gitignorePath.EnsureDir()
```

Comment:
I think you want `FileExists` if you are testing for the existence of the file. `EnsureDir` checks that the directory containing the given path exists.

But, I don't think we should error if we don't find the `.gitignore` file, we should instead create it so that we can add `.turbo` to it